packages/wonder-blocks-core/util/add-style.js

```diff
@@ -7,7 +7,7 @@ import {processStyleList} from "./util.js";
 import type {StyleType} from "./types.js";
 
 // TODO(kevinb): have an a version which uses exact object types
-export default function addStyle<T: React.AbstractComponent<*> | string>(
+export default function addStyle<T: React.AbstractComponent<any> | string>(
```

Comment:
Is it strictly necessary to use `any` here? what would happen if you use `React.Node` instead?

Reply:
`React.Node` isn't a component.

Reply:
I wonder if this should be `mixed` and not `any`.

Reply:
That didn't work.  I need to figure out how to make this be a subtype of `{}`.  In the meantime though `any` isn't any worse than `*`.